core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -194,7 +198,7 @@ object ConsoleConsumer extends Logging {
       .withRequiredArg
       .describedAs("topic")
       .ofType(classOf[String])
-    val whitelistOpt = parser.accepts("whitelist", "Whitelist of topics to include for consumption.")
+    val whitelistOpt = parser.accepts("whitelist", "Regex Pattern, to whitelist all topics matching it, to include for consumption.")
```

Comment:
How about "a regular expression specifying a topic whitelist"

Reply:
"A topic"? It includes all topics to whitelist satisfying the expression.
Changed it to "Regular expression specifying whitelist of topics to include for consumption". Is that fine?